Signal/src/AppDelegate.m

```diff
@@ -300,7 +295,6 @@ - (void)applicationWillResignActive:(UIApplication *)application {
           dispatch_sync(dispatch_get_main_queue(), ^{
               [self protectScreen];
               [[[Environment getCurrent] signalsViewController] updateInboxCountLabel];
-              [TSSocketManager resignActivity];
```

Comment:
`TSSocketManager` now observes application state and registration state events itself, so it doesn't need to be poked by the app delegate.